system/jlib/jsecrets.cpp

```diff
@@ -797,10 +797,85 @@ IPropertyTree *queryMtlsSecretInfo(const char *name)
             if (ca)
                 ca->setProp("@path", filepath.str());
         }
+        // TLS TODO: do we want to always require verify, even if no ca ?
         verify->setPropBool("@enable", true);
         verify->setPropBool("@address_match", false);
         verify->setPropBool("@accept_selfsigned", false);
         verify->setProp("trusted_peers", "anyone");
     }
     return info;
 }
+
+enum UseMTLS { UNINIT, DISABLED, ENABLED };
+static UseMTLS useMtls = UNINIT;
+
+static CriticalSection queryMtlsCS;
+
+jlib_decl bool queryMtls()
+{
+    CriticalBlock block(queryMtlsCS);
+    if (useMtls == UNINIT)
+    {
+        useMtls = DISABLED;
+#if defined(_USE_OPENSSL)
+# ifdef _CONTAINERIZED
+        IPropertyTree &global = queryGlobalConfig();
+        if (global)
```

Comment:
Just spotted this test is redundant - it returns a reference.  I'm not quite sure what this test would do, or why it even compiles.

Reply:
ok, yes, will remove test.